### PR TITLE
[DM-9914] Update Logstash configuration with @athornton's filebeat additions.

### PR DESCRIPTION
**Pull request by jmatt**
* Re-encrypt vaults using new password. Added password to 1password.
  * Add 011-input-filebeats.conf.j2 template for @athornton's Kubernetes filebeat.
  * Add TLS PKI secrets to group_vars/lfr/vault.
  * Add small Elasticsearch configuration changes between 5.0 and 5.2.

	modified:   requirements.yml
	modified:   .gitignore
	modified:   group_vars/es-master/vault
	modified:   group_vars/kibana/vault
	new file:   group_vars/lfr/vars
	new file:   group_vars/lfr/vault
	modified:   group_vars/vagrant/vars
	modified:   group_vars/vagrant/vault
	modified:   roles/elasticsearch-config/tasks/elasticsearch.yml
	modified:   roles/kibana-config/tasks/main.yml
	modified:   roles/logstash-config/tasks/main.yml
	modified:   roles/logstash-config/templates/010-input-beats.conf.j2
	new file:   roles/logstash-config/templates/011-input-filebeats.conf.j2
	deleted:    roles/logstash-config/templates/019-output-beats.conf.j2
	modified:   roles/logstash-config/templates/019-output-elasticsearch.conf.j2

### CODE REVIEW

File: roles/logstash-config/tasks/main.yml [hunk 1]
```diff
@@ -1,13 +1,39 @@
...
     dest: "/etc/logstash/conf.d/{{ item.file_name }}"
+    
   with_items:
     - {template_name: 010-input-beats.conf.j2,
        file_name: 010-input-beats.conf}
+    - {template_name: 011-input-filebeats.conf.j2,
+       file_name: 011-input-filebeats.conf}
     - {template_name: 019-output-elasticsearch.conf.j2,
        file_name: 019-output-elasticsearch.conf}
+#    - {template_name: 030-input-conda.lsst.codes-s3.conf.j2,
```
Comment (athornton) [inline roles/logstash-config/tasks/main.yml:16]: Not sure you need to leave the comments in.

File: roles/logstash-config/templates/019-output-elasticsearch.conf.j2 [hunk 2]
```diff
@@ -1,7 +1,7 @@
-output {  
+output {
   elasticsearch {
     hosts => ["es.lsst.codes:9200", "es-3.lsst.codes:9200"]
-    sniffing => true
+    sniffing => false
```
Comment (athornton) [inline roles/logstash-config/templates/019-output-elasticsearch.conf.j2:4]: I take it sniffing, since our hosts are pretty static, didn't help anything?

**Review by athornton — APPROVED** [review]
Seems pretty straightforward.

File: roles/logstash-config/templates/019-output-elasticsearch.conf.j2 (hunk 2 above)
Comment (jmatt) [reply roles/logstash-config/templates/019-output-elasticsearch.conf.j2:4]: Yeah sniff will get all ES nodes, then potentially error if they aren't reachable.

https://www.elastic.co/guide/en/logstash/current/plugins-outputs-elasticsearch.html#plugins-outputs-elasticsearch-sniffing

**Pull request merged**